database/cdb.ksy

```diff
@@ -0,0 +1,76 @@
+meta:
+  id: cdb
+  file-extension: cdb
+  endian: le
+  encoding: ASCII
```

Comment:
There are no strings in this format, so there's nothing to declare encoding for.

Reply:
In 'record' type. Lines 57 and 60